Program3.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+public class Program3 {
+	 public static void main(String[] args) {
+		 String[] names = { "Elena", "Thomas", "Hamilton", "Suzie", "Phil", "Matt", "Alex","Emma", "John", "James", "Jane", "Emily", "Daniel", "Neda", "Aaron", "Kate" }; 
+		 int[] time = { 341, 273, 278, 329, 445, 402, 388, 275, 243, 334, 412, 393, 299, 343, 317, 265 };
+	    
+		 boolean isSorted = false;
+	        int buf;
+	        String name;
+	        while(!isSorted) {
```

Comment:
this works, but algorithm complexity is very high.
read https://javarush.ru/groups/posts/1997-algoritmih-sortirovki-v-teorii-i-na-praktike about Insertion Sort